### ISSUE TITLE
Can't add audio files from Google Drive

### ISSUE THREAD

**Issue by 79goat5** [googledrive/drive-music-player#165]
Using the most recent version of Chrome on Windows 7 and OS X Yosemite, I can't add audio files from Google Drive. When browsing my Drive, Drive Player launches correctly when I double-click an mp3, but when I try to add files to the queue or to make a play list, I can't. I'm prompted to log in, and I do, but I either get the same login screen over and over OR I'm told the feature I'm requesting isn't available at the time and to try again later. This has always been the case. Are there Chrome extensions known to interfere with Drive Player?

**Comment by RanchLady** [comment]
I am having the exact same problems as listed above, does anyone have any suggestions??

**Comment by sam2145** [comment]
nope, this is happening to me too!

**Comment by sam2145** [comment]
.

**Comment by 79goat5** [comment]
Five and a half months later, sounds like it's still an issue.

**Comment by sam2145** [comment]
yep, it seems like the don't care, or don't pay attention to problems
anymore.

**Comment by nedi911** [comment]
Same problem, no solution I take it?

**Comment by 79goat5** [comment]
None that I'm aware of yet.

**Comment by sam2145** [comment]
not yet, it makes me angry
